---
std_msgs/Float32MultiArray bin_locations
